pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -200,8 +201,9 @@ var _ = Describe("Generating fluentd config", func() {
 				path "/var/log/containers/*.log"
 				exclude_path ["/var/log/containers/fluentd-*_openshift-logging_*.log", "/var/log/containers/elasticsearch-*_openshift-logging_*.log", "/var/log/containers/kibana-*_openshift-logging_*.log"]
 				pos_file "/var/log/es-containers.log.pos"
+                                pos_file_compaction_interval 5m
 				refresh_interval 5
-				rotate_wait 5
+				rotate_wait 1
```

Comment:
does this wait 1 sec before checking if the files were rotated?

Reply:
its how long it waits before switch off to the new file waiting for the process to flush its buffer.  I don't see how we would ever need to wati 5s before the kublet flushes since we know fluent cant read fast enough as is

https://docs.fluentd.org/input/tail#rotate_wait